ros_buildfarm/jenkins.py

```diff
@@ -270,8 +270,8 @@ def _diff_configs(remote_config, new_config):
     if ElementTree.tostring(remote_root) == ElementTree.tostring(new_root):
         return []
 
-    xml1 = ElementTree.tostring(remote_root, encoding='unicode')
-    xml2 = ElementTree.tostring(new_root, encoding='unicode')
+    xml1 = ElementTree.tostring(remote_root)
+    xml2 = ElementTree.tostring(new_root)
```

Comment:
Can you please explain why you changed this? Your commit message says:

> unicode is no longer accepted as an option in ElementTree.tostring

But the docs say something different: https://docs.python.org/3/library/xml.etree.elementtree.html#xml.etree.ElementTree.tostring

> Use encoding="unicode" to generate a Unicode string (otherwise, a bytestring is generated).